sdk/cognitivelanguage/azure-ai-language-conversations/swagger/README.md

```diff
@@ -140,7 +140,16 @@ directive:
           $["operationId"] = "AnalyzeConversation";
 ```
 
-```yaml $(tag) == 'release_runtime_1_0'
+```yaml $(tag) == 'release_runtime_1_1_preview'
+# Rename Runtime client async operation
+directive:
+    - from: swagger-document
+      where: $["paths"]["/analyze-conversations/jobs"]["post"]
+      transform: >
+          $["operationId"] = "ConversationAnalysis";
```

Comment:
Interesting. You don't need to include the "method name" e.g. `ConversationAnalysis_Foo`?

Reply:
No - ConversationAnalysis _is_ the method name (for now) as this generates as `begin_conversation_analysis`. In Python DPG, if there is something before the `_` then all operations get put under an operation group. We don't want that or else user code will have to look something like this: 

```
client.operation_group.analyze_conversation(...)
```